src/app/preferences/general-preferences/general-preferences.module.ts

```diff
@@ -0,0 +1,12 @@
+/// <reference path="general-preferences.controller.ts"/>
+/// <reference path="general-preferences.service.ts"/>
+
+namespace Core {
+
+    export const generalPreferencesModule = angular
+      .module('hawtio-general-preferences', [])
+      .controller('PreferencesGeneralController', GeneralPreferencesController)
+      .service('GeneralPreferencesService', GeneralPreferencesService)
```

Comment:
Start from lower case for the service name: `'GeneralPreferencesService'` -> `'generalPreferencesService'`